server/repository/repository.proto

```diff
@@ -44,6 +44,28 @@ message RepoQuery {
 	string repo = 1;
 }
 
+// RepoAccessQuery is a query for checking access to a repo
+message RepoAccessQuery {
+	// The URL to the repo
+	string repo = 1;
+	// Username for accessing repo
+	string username = 2;
+	// Password for accessing repo
+	string password = 3;
+	// Private key data for accessing SSH repository
+	string PrivateKey = 4;
```

Comment:
We call this `SSHPrivateKey` elsewhere. What's with the capitalization of these fields?